build.gradle

```diff
@@ -384,6 +386,26 @@ subprojects {
         }
     }
 
+    dependencyCheck {
+        scanConfigurations = [
+            'compile',
+            'implementation',
+            'compileOnly',
+            'compileClasspath',
+            'runtime',
+            'runtimeOnly',
+            'runtimeClasspath',
+            'default',
+        ]
+        analyzers {
+            assemblyEnabled = false
+        }
+        suppressionFile = file("${rootDir}/OWASP-Suppression.xml")
```

Comment:
I feel like this should be somewhere other than the root. maybe in the `repose-aggregator\src\config` directory

Reply:
Easy enough. I'll go ahead and move it there assuming @wdschei has no issue with it.

Reply:
I'm good with it.
That was just the easiest place to put it for the original prototype.